src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -20,10 +20,7 @@ const sizes = new Map([
 ])
 
 const getDefaultFontIcon = (iconName: string) => {
```

Comment:
it is better to explicitly specify the return type - otherwise we will lose this knowledge